addons/stock_delivery/views/delivery_view.xml

```diff
@@ -89,10 +89,10 @@
             <field name="arch" type="xml">
                 <field name="company_id" position="before">
                     <label for="shipping_weight"/>
-                    <div class="o_row" name="Shipping Weight">
-                        <field name="shipping_weight" class="oe_inline"/>
-                        <span><field name="weight_uom_name"/></span>
-                        <span class="text-muted">(computed: <field name="weight" class="oe_inline" nolabel="1"/></span><span class="text-muted"><field name="weight_uom_name" nolabel="1" class="oe_inline"/>)</span>
+                    <div class="row" name="Shipping Weight">
+                        <div class="col-2"><field name="shipping_weight" class="oe_inline"/></div>
+                        <div class="col-6"><span><field name="weight_uom_name" class="oe_inline"/></span>
+                        <span class="text-muted"> (computed: <field name="weight" class="oe_inline" nolabel="1"/></span><span class="text-muted"><field name="weight_uom_name" nolabel="1" class="oe_inline"/>)</span></div>
```

Comment:
- Remove not needed `oe_inline`
- Respect linebreak and indentation
- Add a light space (`ms-1`) between computed weight and its UoM
```suggestion
                        <div class="col-2"><field name="shipping_weight"/></div>
                        <div class="col-6">
                            <field name="weight_uom_name" class="oe_inline"/>
                            <span class="text-muted">
                                (computed: <field name="weight" class="oe_inline"/>
                                <field name="weight_uom_name" class="ms-1 oe_inline"/>)
                            </span>
                        </div>
```